examples/package.yaml

```diff
@@ -1 +1,17 @@
+%YAML 1.2
+---
 # Toit Package File.
+# TODO:
+# - Clear the example from comments.
+
+# name: YOUR_PACKAGE_NAME
```

Comment:
These comments should be in the package.yaml of the root.
The example's package.yaml file should almost never be modified by hand.